src/center-common/db/migrate/20200902142050_baseline.rb

```diff
@@ -398,21 +382,6 @@ def up_baseline
         END;
         $$;
 
-        CREATE FUNCTION fix_sequence(schema_name text)
-          RETURNS void
-          LANGUAGE plpgsql
-          AS $$
-        BEGIN
-          IF exists(SELECT 1 FROM pg_extension WHERE extname='bdr') THEN
-            RAISE NOTICE 'BDR';
-            PERFORM bdr.bdr_replicate_ddl_command(format('select %I._fix_sequence_all(%L);', schema_name, schema_name));
-          ELSE
-            RAISE NOTICE 'nonBDR';
-            PERFORM _fix_sequence_all(schema_name);
```

Comment:
also the related function _fix_sequence_all(text) becomes unnecessary

Reply:
Removed in https://github.com/nordic-institute/X-Road/pull/1017/commits/e5910cd34d8445f5fd26dcad03775c3e1dc1f370